topic_folders/governance/committee-policy.md

```diff
@@ -1,77 +1,102 @@
+- Meetings and communication:
+  - Organise regular meetings (at least twice a year) and post meeting notes in a consistent location (meeting notes should be public unless the work of
+    the committee requires them to remain private)
+  - Post meetings in the community calendar (note that meetings do not necessarily need to be open to public to attend - for example, the Code of Conduct Committee's 
+business meetings are public but the incident meetings are not)
+  - Aim to host at least one community discussion per year to let the community know about the committee's activities
+  - Utilise other [The Carpentries community communication pathways](https://docs.carpentries.org/topic_folders/communications/index.html) to engage with the community as needed
+- Prepare written reports to the Executive Council with an agreed frequency, e.g. quarterly, biannually or annually (the frequency of reporting can be adjusted by the Executive Council during the lifetime of the committee)
+
+#### Public Documentation
+Each committee should have a public page with the following information:
+
+- The name of the committee and a short description of what the committee does, including its aims and objectives as outlined in the [committee proposal](#committee-proposal)
+- Contact information for the committee
+- Link to the committee's "Roles and Responsibilities" document
+- Link to the committee's other public materials (if available) - e.g. the committee’s GitHub repository, the relevant section of the 
+Handbook, etc. 
+- Information on when the committee meetings are held
+- Link to the public meeting notes (if available)
+- Names of the current and past members including their roles
+- Name of the Core Team liaison person
+- Information on how to apply to join the committee
+
+The committee chair is responsible for keeping the public documentation up to date.
+
+#### Actions Requiring Executive Council Approval
+A committee will not undertake any of the following actions without obtaining approval of the Executive Council:
+
+- Any modifications to the Code of Conduct
+- Creation and/or release of position statements
+- Host Carpentries-branded events
+- Any modifications of The Carpentries logos (refer to the [Carpentries Logo Usage Policy](https://docs.carpentries.org/topic_folders/communications/resources/logos.html))
+- Launch an own fundraising campaign
+
+Note that the above list is not exhaustive. In the case of any ambiguity, committees should consult the Core Team (in the first instance) or the 
+Executive Council to clarify if things they plan to do require approval.
+
+### Committee Termination
+A committee may naturally cease to function upon fulfilling its role, in which case the Executive Council 
+should be notified. The Core Team may terminate committees that it had approved. 
+Executive Council may terminate any committee at any time.
+
+### Past & Current Committees
+
+The following is a list of some community committees:
```

Comment:
some or all?

Reply:
Put 'some' as the page may get outdated and not list them all